cardano-cli/test/cardano-cli-golden/files/golden/conway/tx-proposal.out.json

```diff
@@ -0,0 +1,70 @@
+{
+    "auxiliary scripts": null,
+    "certificates": null,
+    "collateral inputs": [],
+    "era": "Conway",
+    "fee": "181517 Lovelace",
+    "governance actions": [
```

Comment:
@CarlosLopezDeLara> since the key `"governance action"` key appears nested in this field, I named this one `"governance actions"` (note the trailing `s`). I'm highlighting this, because during our last pair programming session, we had said we would use `governance action` in this position.